packages/firestore/src/core/sync_engine.ts

```diff
@@ -343,6 +343,25 @@ export class SyncEngine implements RemoteSyncer {
     });
   }
 
+  /**
+   * Applies an OnlineState change to the sync engine and notifies any views of
+   * the change.
+   */
+  applyOnlineStateChange(onlineState: OnlineState) {
+    const newViewSnapshots = [] as ViewSnapshot[];
+    this.queryViewsByQuery.forEach((_, queryView) => {
```

Comment:
Google's typescript style disallows `_` as an identifier. I don't know if we're still holding ourselves to it, but for better or worse it's a way to pre-settle arguments about these kinds of low-level things. The suggestion on typescript-users was to name unused parameters with their regular name, or if the body is long enough to call out that they're unused with an "unused" prefix or suffix.

Reply:
Oh! I didn't realize that. Fixed, thanks.